src/spikeinterface/widgets/driftmap.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import annotations
+
+import numpy as np
+
+from .base import BaseWidget, to_attr
+
+
+class DriftMapWidget(BaseWidget):
+    """
+    Plot the a drift map from a motion info dictionary.
```

Comment:
Maybe a little more detail on the drift-map concept? (or, maybe this can go in a 'Notes'). "The drift map displays shows the vertical position of detected peaks as a function of recording time. The y-axis is probe depth and the x-axis is time."

Reply:
good point! Adding it!